### PR TITLE
Text Sets: Section header

### PR DESCRIPTION
**Pull request by dmmulroy**
## Summary

Update the Section header Text Sets

## User-facing changes

Text/Sizing changes

## Testing Instructions

Verify that the Section header text set matches [figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=2%3A34107)

---

Partial for #4903

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json [hunk 1]
```diff
@@ -716,17 +716,17 @@
             "vertical": 0
           },
           "type": "text",
-          "content": "The aesthetic quality of a product is integral to its usefulness because products we use every day affect our person and our well-being. But only well-executed objects can be beautiful.",
+          "content": "San Francisco is only seven by seven miles, but its packed of distinct neighborhoods. Stay in the Haight Asbury to taste the Summer of Love, or book off Columbus street for a culinary collision between Little Italy and Chinatown. ",
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json]:
```suggestion
          "content": "San Francisco is only seven by seven miles, but it's packed of distinct neighborhoods. Stay in the Haight Asbury to taste the Summer of Love, or book off Columbus street for a culinary collision between Little Italy and Chinatown. ",
```

think this should be but it IS packed of distinct neighborhoods, so added an apostrophe

File: assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json [hunk 2]
```diff
@@ -1794,17 +1802,18 @@
             "vertical": 0
           },
           "type": "text",
-          "content": "The aesthetic quality of a product is integral to its usefulness because products we use every day affect our person and our well-being. But only well-executed objects can be beautiful.",
+          "content": "Coffee changed everything! Humans began thinking and gathering. It's said that the French and American revolutio ns were planned in coffeehouses. It also gave great thinkers fuel to kick off the industrial revolution.",
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json]:
```suggestion
          "content": "Coffee changed everything! Humans began thinking and gathering. It's said that the French and American revolutions were planned in coffeehouses. It also gave great thinkers fuel to kick off the industrial revolution.",
```

File: assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json [hunk 3]
```diff
@@ -1990,17 +1925,17 @@
             "vertical": 0
           },
           "type": "text",
-          "content": "The aesthetic quality of a product is integral to its usefulness because products we use every day affect our person and our well-being. But only well-executed objects can be beautiful.",
+          "content": "Making time for fun at work brings teams closer, increases creative colaborations, and reduces stress. Use personality tools to understand your teams dynamic and research related activities, such as a paint night for creatives or playing kickball with sport lovers.",
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json]:
```suggestion
          "content": "Making time for fun at work brings teams closer, increases creative collaborations, and reduces stress. Use personality tools to understand your teams dynamic and research related activities, such as a paint night for creatives or playing kickball with sport lovers.",
```

**Review by BrittanyIRL — APPROVED** [review]
🚀 🚢  Looks good!

**Review by carloskelly13 — APPROVED** [review]

**Pull request merged**